Automattic/wp-calypso
# Add product options display names in the product selector block

#### Changes proposed in this Pull Request

* Add product options display names in the product selector block
* Rename `optionNames` to `optionShortNames`

![Screenshot 2019-10-29 at 09 49 29](https://user-images.githubusercontent.com/478735/67751458-71bc0600-fa31-11e9-8430-f8d4b83774f4.png)


#### Testing instructions

* Checkout this branch.
* Go to http://calypso.localhost:3000/plans/ and test the following cases
* Select a site that has no Jetpack Backup product purchased. Confirm the card title is "Jetpack Backup"
* Select a site that has Daily Backup (or purchase this product). Confirm that the card title is now "Jetpack Backup *Daily*". 
* Select a site that has Real-Time Backup (or purchase this product). Confirm that the card title is now "Jetpack Backup *Real-Time*". 

Fixes n/a
